From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: ViKing <viking.yfy@qq.com>
Date: Sat, 29 Aug 2026 12:32:00 +0800
Subject: [PATCH] net: dsa: qca8k: update RE-CS-08 QCA4B VP marker

Give the post-runtime-validation image a distinct temporary marker so
serial and SSH logs can distinguish it from the first QCA4B VP build.

Signed-off-by: ViKing <viking.yfy@qq.com>
---
 drivers/net/dsa/qca/qca8k-8xxx.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/dsa/qca/qca8k-8xxx.c b/drivers/net/dsa/qca/qca8k-8xxx.c
index 9b31efe..b635102 100644
--- a/drivers/net/dsa/qca/qca8k-8xxx.c
+++ b/drivers/net/dsa/qca/qca8k-8xxx.c
@@ -2383,7 +2383,7 @@ qca8k_sw_probe(struct mdio_device *mdiodev)
 	priv->info = of_device_get_match_data(priv->dev);
 	if (qca8k_is_qca8386(priv))
 		dev_info_once(priv->dev,
-			      "RE-CS-08 firmware debug tag: 20260829-103452-qca8386-qca4b-vp-r1\n");
+			      "RE-CS-08 firmware debug tag: 20260829-123200-qca8386-qca4b-vp-r2\n");
 
 	priv->reset_gpio = devm_gpiod_get_optional(priv->dev, "reset",
 						   GPIOD_OUT_HIGH);
